libs/platform/src/lib/components/table/table.component.ts

```diff
@@ -244,9 +244,9 @@ export class TableComponent<T = any> extends Table implements AfterViewInit, OnD
     @Input()
     relationKey: string;
 
-    /** Whether table row has navigation button. */
+    /** Whether row is navigatable. */
     @Input()
-    navigationButton = false;
+    rowNavigatable: string | boolean;
```

Comment:
what does `string` mean here?
could you please add description for it

Reply:
Updated typedoc comment:

```
/** Whether row is navigatable. 
      * Pass boolean value to set state for the all rows.
      * Pass string value with the key of the row item's field to compute state for every single row.
      */
      ```